Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -180,53 +190,46 @@ public function storeOutput()
     }
 
     /**
-     * Archive data directory and save it to Storage, do not actually run the container.
-     * @param array $tags Arbitrary storage tags
+     * Archive data directory and save it to Storage
+     * @param $fileName
+     * @param array $tags
      */
-    public function storeDataArchive(array $tags)
+    public function storeDataArchive($fileName, array $tags)
```

Comment:
kacířská myšlenka - co kdyby tohle provedl kontejner?

snažím se přemejšlet nad architekturou znovupoužitelnou libovolně do budoucna

Reply:
to tezko, protoze je tam mj. stateful output filter, tohle neni znovupouzitelny nekde jinde